accounts/models.py

```diff
@@ -267,24 +267,29 @@ def get_user_tags(self, use_solr=True):
                  ORDER BY tags_tag.name;""" % self.user_id)
 
     def can_post_in_forum(self):
+
+        # A forum moderator can always post
+        if self.user.has_perm('forum.can_moderate_forum'):
+            return True, ""
+
         user_has_posts_pending_to_moderate = self.user.posts.filter(moderation_state="NM").count() > 0
         if user_has_posts_pending_to_moderate:
             return False, "We're sorry but you can't post to the forum because you have previous posts still " \
                           "pending to moderate"
 
-        if self.user.posts.all().count() >= 1 and self.user.sounds.all().count() == 0:
+        if self.num_posts >= 1 and self.num_sounds == 0:
             today = datetime.datetime.today()
             reference_date = self.user.posts.all()[0].created
 
             # Do not allow posts if last post is not older than 5 minutes
             seconds_per_post = settings.LAST_FORUM_POST_MINIMUM_TIME
-            if (today - self.user.posts.all().reverse()[0].created).seconds < seconds_per_post:
+            if (today - self.user.posts.all().reverse()[0].created).total_seconds() < seconds_per_post:
                 return False, "We're sorry but you can't post to the forum because your last post was less than 5 " \
                               "minutes ago"
 
-            # Do not allow posts if user has already posyted N posts that day
+            # Do not allow posts if user has already posted N posts that day
             max_posts_per_day = settings.BASE_MAX_POSTS_PER_DAY + pow((today - reference_date).days, 2)
-            if self.user.posts.filter(created__range=(today-datetime.timedelta(days=1), today)).count() > \
+            if self.user.posts.filter(created__range=(today-datetime.timedelta(days=1), today)).count() >= \
```

Comment:
I made this change because it makes more sense in tests - if the limit is 9 posts, then I want the user to be able to make 9 and then fail on the 10th. With only `>` it would allow 10 and then fail on 11